temperature.js

```diff
@@ -0,0 +1,16 @@
+let Temperature = {
+  toFahrenheit (celsius) {
+    return celsius * (9/5) + 32
+  },
+  toCelsius (fahrenheit) {
+    return (fahrenheit - 32) * (5/9)
+  }
+}
+
+let tempToday = 82 // In Fahrenheit
+
+let tempInCelsius = Math.round(Temperature.toCelsius(tempToday))
+
+let message = `Today's temperature is ${tempToday} °F, which is ${tempInCelsius} °C.`

```

Comment:
What is with all these `let`s? Are you planning on changing what the variables point to?